device/cig/x86_64-cig_cs6436_56p-r0/Cig-CS6436-56P/qos.json.j2

```diff
@@ -0,0 +1,136 @@
+{
+    "TC_TO_PRIORITY_GROUP_MAP": {
+        "AZURE": {
+            "0": "1",
+            "1": "0",
+            "3": "3",
+            "4": "4"
+        }
+    },
+    "MAP_PFC_PRIORITY_TO_QUEUE": {
+        "AZURE": {
+            "3": "3",
+            "4": "4"
```

Comment:
change to , check other qos_config.j2 

{%- include 'qos_config.j2' %}

Reply:
OK, I've updated it.